src/ring_theory/noetherian.lean

```diff
@@ -663,6 +663,11 @@ begin
   refine (submodule.restrict_scalars_embedding R S M).dual.well_founded h
 end
 
+instance ideal.quotient.is_noetherian_ring {R : Type*} [comm_ring R] [h : is_noetherian_ring R]
```

Comment:
Why is this an instance yet `is_noetherian_of_quotient_of_noetherian` is a `theorem`?

Reply:
And can you make the `R` and `M` arguments to `is_noetherian_of_quotient_of_noetherian` implicit to save some underscores in this proof? They're both obviously unified from `N`.

Reply:
I haven't found a reason why `is_noetherian_of_quotient_of_noetherian` should not be an instance, I'll just make it so.